app/jobs/middleware/job_raven_reporter_middleware.rb

```
@@ -0,0 +1,9 @@
+class JobRavenReporterMiddleware
+  def call(_worker, _job, _queue)
+    begin
+      yield
+    rescue => ex
+      Raven.capture_exception(ex)
```

This will swallow the error right? I think you need to re `raise` after this line